tests/integration/ethernet_mtu_test.py

```diff
@@ -102,3 +102,17 @@ def test_decrease_to_ipv6_min_ethernet_frame_size_iface_mtu():
     netapplier.apply(copy.deepcopy(desired_state))
 
     assertlib.assert_state(desired_state)
+
+
+def test_decrease_to_lower_than_min_ipv6_iface_mtu():
+    desired_state = statelib.show_only(('eth1',))
+    origin_desired_state = copy.deepcopy(desired_state)
+    eth1_desired_state = desired_state[INTERFACES][0]
+    eth1_desired_state['mtu'] = 68
```

Comment:
The MTU value should be motivated. 1279 would be enough to be lower than the IPv6 minimum mtu.